src/neo-vm/Unsafe.cs

```diff
@@ -45,8 +45,8 @@ public static void MemoryCopy(byte[] src, int srcOffset, byte[] dst, int dstOffs
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool NotZero(byte[] x)
         {
-            if (x is null)
-                throw new ArgumentNullException(nameof(x));
+            if (x is null) throw new ArgumentNullException(nameof(x));
+
```

Comment:
Unnecessary change.

Reply:
otherwise i need to do a specific test for hit this line